repository/Seaside-Ajaxifier-Core.package/WAAjaxifierLibrary.class/properties.json

```diff
@@ -1,11 +1,11 @@
 {
-	"commentStamp" : "",
+	"commentStamp" : "pmm 8/25/2022 15:03",
 	"super" : "WAFileLibrary",
-	"category" : "JQuery-Core-Libraries",
+	"category" : "Seaside-Ajaxifier-Core",
```

Comment:
Could also be `Javascript-Ajaxifier-Core`, let me know what you think.

Reply:
That's fine.